src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -449,6 +450,87 @@ public void It_publishes_on_release_if_PublishRelease_property_set()
             Assert.True(File.Exists(expectedAssetPath));
         }
 
+
+        [Fact]
+        public void It_publishes_on_release_if_PublishRelease_property_set_in_sln()
```

Comment:
We should add tests that do `dotnet publish --configuration Debug` on a project / solution with `PublishRelease` set to true.  In that case it should publish with the Debug configuration.

Also we should test `dotnet publish -p:Configuration=Debug`, that should also use the Debug configuration but I don't think it currently will.

Reply:
Added tests for both of those situations :)